### PR TITLE
refactor: footer navbar bg

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Update footer navbar for mobile ux background.
- Figma link: https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/daily.dev---Dev-Mode?node-id=7191%3A25201&mode=dev

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-279 #done

### Preview domain
https://mi-279-background-update.preview.app.daily.dev

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]

File: packages/webapp/components/FooterNavBar.tsx [hunk 1]
```diff
@@ -72,13 +72,10 @@ export default function FooterNavBar({
       className={classNames(
         'fixed !bottom-0 left-0 z-2 w-full',
         isNewMobileLayout
-          ? 'footer-navbar bg-gradient-to-t from-blur-baseline via-blur-bg via-70% to-transparent px-2 pt-2'
+          ? 'footer-navbar bg-gradient-to-t from-background-subtle via-background-subtle via-70% to-transparent px-2 pt-2'
```
Comment (kopancek) [inline packages/webapp/components/FooterNavBar.tsx:75]: Should we also change the gradient stop positions? I remember they were defined differently in figma...

Comment (sshanzel) [reply packages/webapp/components/FooterNavBar.tsx:75]: The stop position was at a different number in Figma, but the issue is that the container in Figma does not compute for the whole container. I adjusted it a bit.

Comment (kopancek) [reply packages/webapp/components/FooterNavBar.tsx:75]: And what is the point of starting with `background-subtle` color, then continuing with `background-subtle` color on the next stop? Can we just add `from-70%` instead?

Comment (sshanzel) [reply packages/webapp/components/FooterNavBar.tsx:75]: Right, yeah I think that's the better approach. Let me check the `from-70%`.

**Pull request merged**